### PR TITLE
Add browser compat for HTML table elements

### PR DESCRIPTION
**Pull request by teoli2003**
Browser compat info for:

&lt;caption&gt;
&lt;col&gt;
&lt;colgroup&gt;
&lt;table&gt;
&lt;tbody&gt;
&lt;td&gt;
&lt;tfoot&gt;
&lt;th&gt;
&lt;thead&gt;
&lt;tr&gt;

### CODE REVIEW

**Comment by teoli2003** [comment]
I just rebased it (trivial rebase, no actual change)

**Comment by teoli2003** [comment]
@Elchi3 I rebased this PR with the new schema

File: html/elements/col.json [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": false,
+                "notes": "See bug <a href='https://bugzil.la/915'>915</a>"
```
Comment (Elchi3) [inline html/elements/col.json]: Please always end notes with a full stop. (Here and elsewhere)

Comment (teoli2003) [reply html/elements/col.json]: Fixed.

**Comment by teoli2003** [comment]
@Elchi3 Comments addressed, trivial rebased done.

File: html/elements/col.json (hunk 1 above)
Comment (teoli2003) [reply html/elements/col.json]: Fixed (all occurences)

**Comment by nvmnghia** [comment]
I know currently `width` is not included [in the spec][1], but is there anywhere we can lobby to de-deprecate this?

  [1]: https://html.spec.whatwg.org/#the-col-element

**Pull request merged**